substrate/client/rpc-servers/src/lib.rs

```diff
@@ -52,6 +52,14 @@ const MEGABYTE: u32 = 1024 * 1024;
 /// Type alias for the JSON-RPC server.
 pub type Server = jsonrpsee::server::ServerHandle;
 
+/// Type to encapsulate the server handle and listening address.
+pub struct ServerAndListenAddress {
```

Comment:
can you rename this new struct `Server` and remove the type-alias `Server` above?

Reply:
Done.